### PR TITLE
updated favicon to match new logo

### PR DESCRIPTION
**Pull request by topialla**
perhaps Anna Gusella should have a look before this is published

### CODE REVIEW

File: iris-client-fe/public/favicon.svg [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   inkscape:export-ydpi="96"
+   inkscape:export-xdpi="96"
+   inkscape:export-filename="C:\Users\topialla\Documents\GitHub\iris-client-frontend\iris-client-fe\public\android-chrome-96x96.png"
```
Comment (bkastl) [inline iris-client-fe/public/favicon.svg]: @topialla Ich hab mir mal erlaubt, die Verzeichnis-Interna deines Filesystems aus den Daten zu entfernen.

Comment (topialla) [reply iris-client-fe/public/favicon.svg]: Danke!

**Review by faversente — APPROVED** [review]
Tested and icon in tabs seems to be changed correctly

**Comment by topialla** [comment]
closes iris-connect/iris-backlog#125

**Pull request merged**